README.md

```diff
@@ -3,25 +3,64 @@
 An open-source interoperability standard for tools and content
 management systems used in media production.
 
-OpenAssetIO defines a common set of interactions between a host of the
-API (eg: a Digital Content Creation tool or pipeline script) and an
-Asset Management System.
-
 It aims to reduce the integration effort and maintenance overhead of
 modern CGI pipelines, and pioneer new, standardized asset-centric
-workflows in post-production tooling.
+workflows in content creation tooling.
+
+## Problem Statement
+
+In modern production pipelines, data is often managed by an
+authoritative system (Asset Management System, Digital Asset Manager,
+MAM, et. al).
+
+It is common for media creation tools to reference managed data by its
+present location in a file system.
+
+This not only prohibits document portability, but strips semantically
+meaningful information about the purpose or history of the data -
+complicating topics such as loading, version management and distributed
+computation.
+
+In addition, file based containers are not suitable representations for
+many common entities managed by these systems.
```

Comment:
[consider] Add an example

> For example a "shot" (in the Media and Entertainment sense) is an entity with properties (such as "frame range") that a DCC tool may wish to query. The source of these properties is ultimately the AMS. Forcing these properties into a bespoke file format, just to make them accessible to the DCC tool, is less than ideal. It would be better if the DCC tool could query the AMS directly.

Reply:
I was conscious about how long the text is - thinking of Wall of Text  Syndrome™ in a README would be really bad. Maybe we need a separate document where we can elaborate on these points?

Reply:
Capturing an off-line chat: The plan is to add a separate `PROBLEM_STATEMENT.md` document that elaborates on these issues specifically in terms of post production within the M&E sector, with concrete examples. Leaving the specific text as an "executive summary", and hopefully avoiding an outfacing amount of copy on the main page. [This commit](https://github.com/OpenAssetIO/OpenAssetIO/pull/559/commits/36919cd9a066089afe94b5dadd005194924ecc45) prunes a little fat and re-phrases more as a summary, with the intent to add the detailed documentation in a follow-up commit.